src/Core/Framework/Framework.php

```diff
@@ -152,16 +149,6 @@ public function boot(): void
             MeterProvider::bindMeter($this->container);
         }
 
-        // @deprecated tag:v6.7.0 - remove complete if condition
-        if (!Feature::isActive('DYNAMIC_ENTITY_EXTENSIONS')) {
```

Comment:
this change requires this "fix": https://github.com/shopware/shopware/pull/6585

otherwise you can't uninstall commercial plugin anymore 🙈

Reply:
also without handling the other use case of the flag the state is kinda broken, should we leave this change out of this PR and discuss separately?

Reply:
removed for now, we can discuss separately